electron_app/src/electron-main.js

```diff
@@ -322,14 +322,13 @@ app.on('ready', () => {
         webPreferences: {
             preload: preloadScript,
             nodeIntegration: false,
-            sandbox: true,
```

Comment:
Removing this sounds slightly scary https://electronjs.org/docs/api/sandbox-option

Is it possible to make this all work with this still enforcing?

Reply:
![image](https://user-images.githubusercontent.com/6655915/60758148-17c3aa80-a014-11e9-8a95-a50611c7413f.png)
After enabling Sandbox errors like this happened. Could you help me figure this out?

Reply:
(With my limited knowledge of Electron - I don't work on Riot) perhaps the sandbox prevents some parts of the code from communicating directly to modules, and instead you need to use IPC communication to wrap calls to the lib.

@dbkr would know more though.

Reply:
ok, I'll try to figure it out today

Reply:
ok I think it's fixed